otel-integration/k8s-helm/values.yaml

```diff
@@ -545,3 +545,90 @@ opentelemetry-cluster-collector:
 
 opentelemetry-agent-windows:
   enabled: false
+opentelemetry-gateway:
+  enabled: false
```

Comment:
Is there a reason to include the values here instead of in the override file, if we're disabling the sub-chart in the default values file?